content/en/security/cloud_security_management/setup/_index.md

```diff
@@ -30,9 +30,15 @@ further_reading:
 
 To get started with Cloud Security Management (CSM), follow these steps:
 
-1. [Enable Agentless Scanning](#enable-agentless-scanning)
-1. [Deploy the Agent for additional coverage](#deploy-the-agent-for-additional-coverage)
-1. [Enable additional features](#enable-additional-features)
+- [Overview](#overview)
```

Comment:
"Overview" and "Further reading" aren't really steps -- you might skip the Further reading link, and just link to the Overview from "Cloud Security Management (CSM)" instead of making it a step.

Reply:
it's a generated TOC that the previous writer added. I'll rephrase the intro to be clearer.